tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/Makefile.inc

```diff
@@ -0,0 +1,8 @@
+ifeq ($(TARGET), zephyr_vexriscv)
```

Comment:
I've customized the build process so this file has to be defined (and it is actually) a bit different. I think we should not merge this file.